apps/ex_nvr/lib/ex_nvr/authorization.ex

```diff
@@ -0,0 +1,10 @@
+defmodule ExNVR.Authorization do
+  @moduledoc """
+  Basic user authorization module by role.
+  """
+  def is_authorized?(:admin, :device, _action), do: true
+  def is_authorized?(:user, :device, :read), do: true
+  def is_authorized?(_role, :recording, :read), do: true
```

Comment:
If you put `?` at the end of the function, there's no need for `is_` prefix. Update the function to accept the whole user struct and not only the `role`.